app/src/main/res/values/integers.xml

```diff
@@ -11,7 +11,6 @@
   <integer name="topic_revision_span_count">2</integer>
   <integer name="ongoing_topics_span_count">2</integer>
   <integer name="completed_story_span_count">2</integer>
-  <integer name="profile_progress_span_count">1</integer>
+  <integer name="profile_progress_span_count">2</integer>
   <integer name="profile_progress_limit">3</integer>
```

Comment:
Shouldn't we update the `profile_progress_limit`?  
In the [mocks](https://xd.adobe.com/view/e8aa4198-3940-47f9-514a-f41cc54457f6-9e9b/screen/7fb5929e-4652-4412-bf8c-64d94f4159f9/) we have 3 items, but is this the maximum numbers of items we can have?

Reply:
Yes that is correct, we can have maximum 3 items here.